src/constructors.jl

```diff
@@ -124,7 +142,7 @@ function check_macro_kwarg(arg, inline::Int8 = Int8(2), U::Int8 = zero(Int8), T:
     @assert arg.head === :(=)
     i = check_inline(arg)
     if i !== nothing
-        inline = i ? Int8(2) : Int8(-1)
+        inline = i ? Int8(2) : Int8(-1)  # what do these constants mean and why not keep true/false?
```

Comment:
There were once three possibilities, now there are two.

Reply:
I see you already did that, many thanks! I don't mind merge commits but I could have also done that. (Easier for me that you did, thanks!)